Bottom: f5cf76248af35ca95b96c66f56eaaaa2318c3b0a
Top:    b134b31ba595f72dde5d1901e03869eb279611ac
Author: Robert Berger <robert.berger@ReliableEmbeddedSystems.com>
Date:   2017-10-26 18:29:25 +0300

poky: LICENSE in packagegroup makes no sense - removed

Signed-off-by: Robert Berger <robert.berger@ReliableEmbeddedSystems.com>


---

diff --git a/meta/recipes-core/packagegroups/nativesdk-packagegroup-sdk-host.bb b/meta/recipes-core/packagegroups/nativesdk-packagegroup-sdk-host.bb
index 2ca6392..aee4a03 100644
--- a/meta/recipes-core/packagegroups/nativesdk-packagegroup-sdk-host.bb
+++ b/meta/recipes-core/packagegroups/nativesdk-packagegroup-sdk-host.bb
@@ -4,7 +4,6 @@
 
 SUMMARY = "Host packages for the standalone SDK or external toolchain"
 PR = "r12"
-LICENSE = "MIT"
 
 inherit packagegroup nativesdk
